shell/packages/sandstorm-ui-grainlist/grainlist-client.js

```diff
@@ -357,6 +357,14 @@ Template.sandstormGrainTable.events({
     }
   },
 
+  "click td.select-grain": function (event, instance) {
```

Comment:
This doesn't seem to have worked AFAICT: clicking just outside the checkbox does not check it for me.